draft-gruessing-moq-requirements.md

```diff
@@ -201,7 +215,7 @@ senders and receivers.
 
 **Senders/Receivers**: One to One
 **Bi-directional**: Yes
-**Latency**: Significantly less than "Ultra-Low"
+**Latency**: Ull100
```

Comment:
Is VC not also ull500 or is 500ms-1s lag acceptable/normal?

Reply:
@afrind - I'm pretty sure that the number is under 500 ms, but a quick Google turns up opinions about 300 ms, and about 200 ms. I'd like to understand what the right categories are, that would cover the use cases we seem to be converging on. So I'll make this change, and keep the PR open.